shuffle/cli/Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.38"
 bcs = "0.1.2"
+dir = "0.1.2"
```

Comment:
This dependency needs to be remove for a couple reasons:
1. If you look at the Cargo.lock file its a very heavy-weight dependency for what you're using it for, determining the home dir for the user
2. the [dir crate](https://crates.io/crates/dir/0.1.2) is licensed as GPL-3.0 which is not compatible with how diem is licensed (Apache 2) 
3. This dependency is hardly used by the community

Reply:
Through some quick searching it seems like the most used crate for this purpose is the [directories](https://crates.io/crates/directories/4.0.1) crate. It has 3 million downloads and has a license compatible with the project

Reply:
Awesome! Will keep those 3 things in mind and look at that directories crate. From now on, I'll double check to see if a crate is Apache 2 licensed

Reply:
I might also argue that there's nothing wrong with either using a temp path or requiring the user to specify where they want things to happen.